### PR TITLE
Update drivebase_odom_publisher.cpp

### PR DESCRIPTION
**Pull request by Jhall118**
v_lin (line 107) and v_ang (line 108) are using tread distance, not speed, so we need to add a time component. 

Updated some constants and added some comments. The turning accuracy is sensitive to wheel base.  Will need to experimentally get it right, because the ideal value is not best for real world performance.  This value currently is off by about 5% when turning.

### CODE REVIEW

File: src/tfr_sensor/src/drivebase_odom_publisher.cpp [hunk 1]
```diff
@@ -62,14 +62,14 @@ class DrivebaseOdometryPublisher
         boost::function<void(const std_msgs::Float64&)> rightTreadCallback = [this](const std_msgs::Float64& msg) {this->rightTreadSpeed = msg.data; };
         
 
-		leftTreadDistanceSub = n.subscribe<std_msgs::Float64>("/left_tread_distance", 15, leftTreadCallback);
+	leftTreadDistanceSub = n.subscribe<std_msgs::Float64>("/left_tread_distance", 15, leftTreadCallback);
```
Comment (apw6) [inline src/tfr_sensor/src/drivebase_odom_publisher.cpp]: indentation

File: src/tfr_sensor/src/drivebase_odom_publisher.cpp [hunk 2]
```diff
@@ -62,14 +62,14 @@ class DrivebaseOdometryPublisher
...
         rightTreadDistanceSub = n.subscribe<std_msgs::Float64>("/right_tread_distance", 15, rightTreadCallback);
         
         //odometry_publisher: publish to the location of the base_footprint tracked by tread motion.
         odometry_publisher = n.advertise<nav_msgs::Odometry>("/drivebase_odom", 15); 
         
         ///set_drivebase_odometry : resets the basis of odometry to a new position
-        set_odometry = n.advertiseService("set_drivebase_odometry", &DrivebaseOdometryPublisher::setOdometry, this);
+        set_odometry = n.aWhdvertiseService("set_drivebase_odometry", &DrivebaseOdometryPublisher::setOdometry, this);
```
Comment (apw6) [inline src/tfr_sensor/src/drivebase_odom_publisher.cpp]: typo

File: src/tfr_sensor/src/drivebase_odom_publisher.cpp [hunk 3]
```diff
@@ -104,11 +104,13 @@ class DrivebaseOdometryPublisher
         }
 
         //basic differential kinematics to get combined velocities
-        double v_ang = (rightTreadSpeed-leftTreadSpeed)/wheel_span;
-        double v_lin = (rightTreadSpeed+leftTreadSpeed)/2;
+ 	double v_right = 1/d_t * rightTreadSpeed; //TODO rename these to TreadDistance instead of TreadSpeed
```
Comment (apw6) [inline src/tfr_sensor/src/drivebase_odom_publisher.cpp]: why not  `double v_right = rightTreadDistance/d_t;`?

Comment (Jhall118) [reply src/tfr_sensor/src/drivebase_odom_publisher.cpp]: Yeah that is a cleaner way

**Pull request closed without merging**